editor/store/actions.js

```diff
@@ -766,3 +767,63 @@ export function unregisterToken( name ) {
 		name,
 	};
 }
+
+export function fetchSharedBlocks( id ) {
+	deprecated( 'fetchSharedBlocks', {
+		alternative: 'fetchReusableBlocks',
+		version: '3.6',
+		plugin: 'Gutenberg',
+	} );
+
+	return fetchReusableBlocks( id );
+}
+
+export function receiveSharedBlocks( results ) {
+	deprecated( 'receiveSharedBlocks', {
+		alternative: 'receiveReusableBlocks',
+		version: '3.6',
+		plugin: 'Gutenberg',
+	} );
+
+	return receiveReusableBlocks( results );
+}
+
+export function saveSharedBlock( id ) {
+	deprecated( 'saveSharedBlock', {
+		alternative: 'saveReusableBlock',
+		version: '3.6',
+		plugin: 'Gutenberg',
+	} );
+
+	return saveReusableBlock( id );
+}
+
+export function deleteSharedBlock( id ) {
+	deprecated( 'deleteSharedBlock', {
+		alternative: 'deleteReusableBlock',
+		version: '3.6',
+		plugin: 'Gutenberg',
+	} );
+
+	return deleteReusableBlock( id );
+}
+
+export function updateSharedBlockTitle( id, title ) {
+	deprecated( 'updateSharedBlockTitle', {
+		alternative: 'updateReusableBlockTitle',
+		version: '3.6',
+		plugin: 'Gutenberg',
+	} );
+
+	return updateReusableBlockTitle( id, title );
+}
+
+export function convertBlockToSaved( clientId ) {
```

Comment:
This function was called `convertBlockToShared`, not `convertBlockToSaved`.

Reply:
🤦‍♂️ good catch